### PR TITLE
Fix breaking typo, some other clean up

### PR DESCRIPTION
**Pull request by burkasaurusrex**

### CODE REVIEW

File: app/imdb_tools.py [hunk 1]
```diff
@@ -86,12 +86,25 @@ def imdb_get_movies(config_path, plex, data):
...
-def tmdb_get_movies(config_path, plex, data, is_list=False):
+def tmdb_parse_id(data):
     try:
-        tmdb_id = re.search('.*?(\\d+)', data)
+        tmdb_id = re.search('.*?(\\d+)', str(data)) # re.search requires a string
         tmdb_id = tmdb_id.group(1)
+        return tmdb_id
     except AttributeError:  # Bad URL Provided
-        raise ValueError("| Config Error: TMDb: {} is invalid".format(data))
+        raise ValueError("| Config Error: TMBd ID: {} is invalid it must be a number".format(data))
```
Comment (meisnate12) [inline app/imdb_tools.py:95]: typo i think this should be TMDb not TMBd

**Review by meisnate12 — APPROVED** [review]
looks good to me just the one typo

**Comment by burkasaurusrex** [comment]
Any other comments? We should merge this ASAP given that TMDb collections fail silently without it

**Comment by meisnate12** [comment]
No looks good let's get this in

**Pull request merged**